Tests/CommandsTests/PackageToolTests.swift

```diff
@@ -1852,6 +1852,109 @@ final class PackageToolTests: CommandsTestCase {
         }
     }
 
+    func testCommandPluginNetworkingPermissions(permissionsManifestFragment: String, permissionError: String, reason: String, remedy: [String]) throws {
+        // Only run the test if the environment in which we're running actually supports Swift concurrency (which the plugin APIs require).
+        try XCTSkipIf(!UserToolchain.default.supportsSwiftConcurrency(), "skipping because test environment doesn't support concurrency")
+
+        try testWithTemporaryDirectory { tmpPath in
+            // Create a sample package with a library target and a plugin.
+            let packageDir = tmpPath.appending(components: "MyPackage")
+            try localFileSystem.writeFileContents(packageDir.appending(components: "Package.swift")) {
+                $0 <<< """
```

Comment:
iirc we have an overload that takes string directly so we dont need to mess with the stream (which we should deprecate)

Reply:
Just following the existing tests here, but I can change it